src/gui/SettingsWidgetGeneral.ui

```diff
@@ -367,6 +367,13 @@
      </widget>
     </widget>
    </item>
+   <item row="15" column="0">
+    <widget class="QCheckBox" name="autoCloseOnScreenLockCheckBox">
```

Comment:
This checkbox is appearing OUTSIDE the basic settings view. I think this should really be under the Security Settings anyway.